src/field_specials.c

```diff
@@ -65,6 +65,7 @@
 #include "constants/weather.h"
 #include "constants/metatile_labels.h"
 #include "palette.h"
+#include "constants/pokemon.h"
```

Comment:
This'll already be included, it's part of `global.h`

Reply:
Corrected :eyes: